src/metabase/events/notification.clj

```diff
@@ -76,6 +76,8 @@
                                                       :notification_ids (map :id notifications)}}
         (log/infof "Found %d notifications for event: %s" (count notifications) topic)
         (doseq [notification notifications]
+          (def notification (assoc notification :payload {:event_info  (maybe-hydrate-event-info topic event-info)
+                                                          :event_topic topic}))
```

Comment:
Def in a loop? 💣 

Assume this was for debugging?

Reply:
:face-palm: